CAP/gap/UniversalObjects.gd

```diff
@@ -1098,6 +1098,9 @@ DeclareOperation( "AddInitialObjectFunctorial",
 DeclareOperationWithCache( "DirectSumOp",
                            [ IsList, IsCapCategoryObject ] );
 
+DeclareOperation( "DirectSumOp",
```

Comment:
Actually you can call `DirectSum( category, list_of_objects )`,
which redirects to `DirectSumOp( list_of_objects, category )`.